yarn-project/package.json

```diff
@@ -51,6 +51,7 @@
     "sequencer-client",
     "scripts",
     "types",
+    "txe",
```

Comment:
Do other packages depend on this? If so please remember to also release this npm package otherwise any dependant packages will break. (there is a deploy_npm script somewhere)

Reply:
Not yet, and I wouldn't encourage people to use TXE yet on their projects until it's a little bit more feature complete

Reply:
Added to deploy_npm.sh!